assets/js/googlesitekit/widgets/register-defaults.js

```diff
@@ -156,6 +157,26 @@ export function registerDefaults( widgetsAPI ) {
 			},
 			CONTEXT_MAIN_DASHBOARD_TRAFFIC
 		);
+
+		widgetsAPI.registerWidget(
+			'audienceConnectAnalyticsCTA',
+			{
+				Component: ConnectAnalyticsCTATileWidget,
+				width: widgetsAPI.WIDGET_WIDTHS.FULL,
+				priority: 1,
+				wrapWidget: false,
+				modules: [ 'analytics-4' ],
+				isActive: ( select ) => {
+					const isAnalyticsConnected =
+						select( CORE_MODULES ).isModuleConnected(
+							'analytics-4'
+						);
+
+					return ! isAnalyticsConnected;
+				},
```

Comment:
IMO, the one-liner seems cleaner. It's just a personal preference, and applying it is unnecessary unless you are okay with it :)
```suggestion
				isActive: ( select ) =>
					! select( CORE_MODULES ).isModuleConnected( 'analytics-4' ),
```

Reply:
This will change in a follow up issue where it will also take the audience setup into consideration, when it will no longer work as a simple one liner, I've added a TODO comment to explain this. Cheers.